interface/patient_file/summary/stats_full.php

```diff
@@ -285,8 +285,8 @@ function newEncounter() {
 
     echo " <tr class='$bgclass detail' $colorstyle>\n";
     echo "  <td style='text-align:left' data-text='$disptitle' class='$click_class' id='$rowid'>" . text($disptitle) . "</td>\n";
-    echo "  <td>" . text(date(DateFormatRead(true), strtotime($row['begdate']))) . "&nbsp;</td>\n";
-    echo "  <td>" . text(date(DateFormatRead(true), strtotime($row['enddate']))) . "&nbsp;</td>\n";
+    echo "  <td>" . text($row['begdate']) . "&nbsp;</td>\n";
+    echo "  <td>" . text($row['enddate']) . "&nbsp;</td>\n";
```

Comment:
This is not the right approach! Just check if the value is `NULL` or `""` then show a default `-`, the functions there are not be removed